### PR TITLE
fix: apiDescriptionDocument input in storybook

### PR DESCRIPTION
**Pull request by mpodlasin**
I've noticed that some fields in storybook are not displayed properly, which makes testing and debugging a bit more difficult (at least for me):

Before:

Screenshot 2021-09-28 at 11 32 53

After:

Screenshot 2021-09-28 at 11 31 02

### CODE REVIEW

**Review by mallachari — APPROVED** [review]

**Review by mmiask — APPROVED** [review]
Good catch!

File: packages/elements/src/containers/API.stories.tsx [hunk 1]
```diff
@@ -42,13 +42,13 @@ APIWithJSONProvidedDirectly.storyName = 'Direct JSON Input (Zoom)';
 
 export const APIWithoutDescription = Template.bind({});
 APIWithoutDescription.args = {
-  apiDescriptionDocument: simpleApiWithoutDescription,
+  apiDescriptionDocument: JSON.stringify(simpleApiWithoutDescription, undefined, 2),
```
Comment (domagojk) [inline packages/elements/src/containers/API.stories.tsx:45]: I don't know why, but I always use `null` for this... but it's the same 🤷‍♂️ 

```suggestion
  apiDescriptionDocument: JSON.stringify(simpleApiWithoutDescription, null, 2),
```

**Review by domagojk — APPROVED** [review]

File: packages/elements/src/containers/API.stories.tsx (hunk 1 above)
Comment (mpodlasin) [reply packages/elements/src/containers/API.stories.tsx:45]: I know, team void crew doesn't like `undefined` 😂

Comment (mpodlasin) [reply packages/elements/src/containers/API.stories.tsx]: In the other place in the file there is also `null` used, so I am changing to `null`, for consistency's sake. You win!

Comment (domagojk) [reply packages/elements/src/containers/API.stories.tsx]: hehe, no I was actually a silent spy regarding `undefined` and secretly changed `void 0` wherever they wouldnt notice 
🤫  🥷

**Pull request merged**